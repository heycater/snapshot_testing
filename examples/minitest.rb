require "minitest/autorun"
require "bundler/setup"
require "snapshot_testing/minitest"

class ExampleTest < Minitest::Test
  prepend SnapshotTesting::Minitest

  def test_snapshot
    assert_snapshot "hello"
    assert_snapshot "goodbye"
  end
end
