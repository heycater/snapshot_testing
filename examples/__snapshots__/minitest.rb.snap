snapshots["test_snapshot 1"] = <<~SNAP
hello
SNAP

snapshots["test_snapshot 2"] = <<~SNAP
goodbye
SNAP
